frontend/src/main/scala/akkaviz/frontend/vis/package.scala

```diff
@@ -0,0 +1,249 @@
+package akkaviz.frontend
+
+import org.scalajs.dom.{Element, Event}
+import org.scalajs.dom.html.Canvas
+
+import scala.scalajs.js
+import scala.scalajs.js.annotation.JSName
+import scala.scalajs.js.|
+
+package object vis {
+
+  @js.native
+  trait Node extends js.Any {
+    def id: js.Any
+
+  }
+
+  object Node {
+    def apply(id: String, label: String, value: js.UndefOr[Int] = js.undefined, color: js.UndefOr[String] = js.undefined): vis.Node = {
+      js.Dynamic.literal(id = id, label = label, value = value, color = color).asInstanceOf[vis.Node]
+    }
+  }
+
+  @js.native
+  trait Edge extends js.Any {
+    def id: js.Any
+  }
+
+  object Edge {
+    def apply(id: String, from: String, to: String, arrows: js.UndefOr[String] = "to", title: js.UndefOr[String] = js.undefined, value: js.UndefOr[Int] = js.undefined): vis.Edge = {
+      js.Dynamic.literal(id = id, from = from, to = to, value = value, arrows = arrows).asInstanceOf[vis.Edge]
+    }
+  }
+
+  @js.native
+  trait DataSetOptions extends js.Any {
+
+  }
+
+  @js.native
+  @JSName("vis.DataSet")
+  class DataSet[T](data: js.Array[T] = js.Array[T](), options: js.UndefOr[DataSetOptions] = js.native) extends js.Any {
+
+    def add(d: T): js.Array[String] = js.native
+
+    def add(d: js.Array[T]): js.Array[String] = js.native
+
+    def remove(d: String): js.Array[String] = js.native
+
+    def remove(d: js.Array[String]): js.Array[String] = js.native
+
+    def get(d: String): T = js.native
+
+    def get(d: js.Array[String]): js.Array[T] = js.native
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Do not prefix getters with 'get'](https://www.codacy.com/app/lustefaniak/akka-viz/file/2159111834/issues/source?bid=3121282&fileBranchId=3158877#l54)